style.css

```diff
@@ -0,0 +1,417 @@
+	display: -ms-flexbox;
+	display: flex;
+	gap: 40px;
+}
+
+.headerBtn {
+	color: #ffffff;
+	text-decoration: none;
+	font-size: 2.6rem;
+	font-weight: normal;
+	border-bottom: 4px solid #ffffff;
+}
+
+.headerBtn:hover {
+	border-bottom: 4px solid #9e381c;
+}
+
+@media only screen and (max-width: 310px) {
+	
+	.headerTitle {
+		font-size: 6.8rem;
+	}
+	
+	.headerText {
+		font-size: 2.6rem;
+	}
+}
+
+/* -- About me -- */
+
+.aboutMe {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-webkit-box-align: center;
+	    -ms-flex-align: center;
+	        align-items: center;
+	background-color: #808080;
+	min-height: 500px;
+	width: 100vw;
+	padding: 40px 10%;
+	-webkit-box-shadow: inset 0px 12px 24px -12px rgba(0, 0, 0, 0.5);
+	box-shadow: inset 0px 12px 24px -12px rgba(0, 0, 0, 0.5);
+}
+
+.aboutMeText {
+	color: #ffffff;
+	-webkit-box-flex: 1;
+	    -ms-flex-positive: 1;
+	        flex-grow: 1;
+}
+
+.aboutMeText h2 {
+	font-size: 3.2rem;
+}
+
+.aboutMeText p {
+	font-size: 2.4rem;
+}
+
+.aboutMeText a {
+	display: inline-block;
+	line-height: 1.2;
+	color: #ffffff;
+	text-decoration: none;
+	border-bottom: 4px solid rgba(158, 56, 28, 0.8);
+}
+
+.aboutMeText a:hover {
+	border-bottom: 4px solid rgba(255, 255, 255, 0.8);
+}
+
+.aboutMeImg {
+	width: 100%;
+	max-height: 400px;
+	max-width: 400px;
+	border-radius: 50%;
+	-webkit-box-flex: 1;
+	    -ms-flex-positive: 1;
+	        flex-grow: 1;
+}
+
+.verticalDivider {
+	width: 2px;
+	background-color: rgba(255,255,255,0.3);
+	height: 400px;
+	max-height: 80%;
+	margin: 40px;
+}
+
+
+@media only screen and (max-width: 1010px) {
+	.aboutMe {
+		-ms-flex-wrap: wrap;
+		    flex-wrap: wrap;
+		-webkit-box-pack: center;
+		    -ms-flex-pack: center;
+		        justify-content: center;
+		gap: 40px;
+	}
+	
+	.verticalDivider {
+		display: none;
+	}
+}
+
+/* -- Projects -- */
+
+.recentProjects {
+	width: 100vw;
+	padding: 40px 10%;
+	background-color: #ffebcd;
+}
+
+.recentProjects h2 {
+	font-size: 3.2rem;
+	margin-bottom: 40px;
+	text-align: center;
+}
+
+.projectGrid {
+	max-width: 100%;
+	display: -ms-grid;
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+	gap: 15px;
+}
+
+.projectItem img {
+	max-width: 100%;
+}
+
+.projectName {
+	font-size: 1.4rem;
+	display: inline-block;
+	line-height: 1.2;
+	color: #000000;
+	text-transform: uppercase;
+	text-decoration: none;
+	border-bottom: 4px solid rgba(158, 56, 28, 0.8);
+	margin-left: 10px;
+}
+
+.projectName:hover {
+	border-bottom: 4px solid rgba(255, 255, 255, 0.8);
+}
+
+.projectDesc {
+	font-size: 1.4rem;
+	margin-left: 10px;
+}
+
+.projectLink {
+	text-decoration: none;
+	color: #000000;
+	margin-left: 10px;
+	font-size: 1.4rem;
+	border-bottom: 4px solid rgba(158, 56, 28, 0.8);
+	display: inline-block;
+	line-height: 1.5;
+}
+
+.projectLink:hover {
+	border-bottom: 4px solid #FFFFFF;
+}
+
+/* -- Contact -- */
+
+.contactMe {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
```

Comment:
not sure any of this flex stuff is helping you, potentially just overcomplicating, can be achieved by using mainly the default styles that the contained elements have

Reply:
Yep. Have removed all flex properties for this